### PR TITLE
Update devDependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by MartijnCuppens** [comment]
Fine by me

**Comment by XhmikosR** [comment]
OK the rename doesn't affect this PR anyway. I'm gonna wait until that is done to merge though.

**Review by Johann-S — APPROVED** [review]

**Pull request merged**
